python/ray/dag/class_node.py

```diff
@@ -246,6 +246,9 @@ def _execute_impl(self, *args, **kwargs):
     def __str__(self) -> str:
         return get_dag_node_str(self, f"{self._method_name}()")
 
+    def __repr__(self) -> str:
```

Comment:
Question: why is this needed?
(__repr__ is not added in other DAGNode)

Reply:
this was to make debugging easy when it was nested inside a list (in this case, repr is used).